### PR TITLE
Internationalization

### PR DESCRIPTION
**Pull request by ericlavigne**
Implements Internationalization, using Spanish as an example, as described in https://github.com/Co-Epi/app-ios/issues/95 .

### CODE REVIEW

File: CoEpi/ui/alerts/AlertsViewModel.swift [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import Dip
+import UIKit
```
Comment (i-schuetz) [inline CoEpi/ui/alerts/AlertsViewModel.swift:2]: For what is `UIKit` needed?

Comment (ericlavigne) [reply CoEpi/ui/alerts/AlertsViewModel.swift:2]: My original approach used a translation function from UIKit. Swiftgen replaced that function, so I will remove the UIKit import.

I'll have instructions added to README soon.

Comment (ericlavigne) [reply CoEpi/ui/alerts/AlertsViewModel.swift]: Fixed both issues.

**Comment by i-schuetz** [comment]
Consider moving the instructions to the wiki and linking from the readme. They're a bit too detailed for the readme.

**Comment by ericlavigne** [comment]
> Consider moving the instructions to the wiki and linking from the readme. They're a bit too detailed for the readme.

I created a new Internationalization wiki page and changed the README to link to that page. Merging now.

**Pull request merged**